docs/Intro/dashboard.md

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 4
+---
+
+# The Chronicle Dashboard
+The [Chronicle Dashboard](https://chroniclelabs.org/dashboard) enables full transparency over where the Oracle-reported data was sourced and who reported/signed it.
+
+## Where is the data sourced from?
+
+Using the dashboard, users can select a time interval on the graph and, for each validator, use the drop-down arrow to see the different data sources that have been used. Typically, the data sources are a mix of on-chain and off-chain data sources with high liquidity.
```

Comment:
on-chain or onchain? Personally prefer onchain (who the fcks write e-mail instead of email?) but we should at least be consistent.